docs/modules/ROOT/pages/concepts.adoc

```diff
@@ -0,0 +1,27 @@
+= Concepts
+
+== Connectors
+
+https://trino.io/docs/current/overview/use-cases.html#what-trino-is[Trino] is a tool designed to efficiently query vast amounts of data using distributed queries. It is not a database with its own store but rather interacts with many types of store. Trino connects to these stores - or data sources - via https://trino.io/docs/current/connector.html[connectors].
+Each connector enables access to a specific underlying datasource such as a Hive warehouse, a PostgreSQL database or a Druid instance.
+
+A Trino cluster comprises two roles: the Coordinator, responsible for managing and monitoring work loads, and the Worker, which is responsible for executing specific tasks that together make up a work load. The workers fetch data from the connectors, execute tasks and share intermediate results. The coordinator collects and consolidates these results for the end-user.
+
+== Catalogs
+
+An instance of a connector is called a catalog.
+Think of a setup containing a large Hive warehouse running on HDFS.
+There may exist two different catalogs called e.g. `warehouse_1` and `warehouse_2` each specifying the same `hive` connector.
+
+Currently, the following connectors are supported:
+
+* https://trino.io/docs/current/connector/hive.html[Hive]
+* https://trino.io/docs/current/connector/iceberg.html[Iceberg]
+* https://trino.io/docs/current/connector/tpcds.html[TPCDS]
+* https://trino.io/docs/current/connector/tpch.html[TPCH]
+
+== Catalog references
+
+Within Stackable a `TrinoCatalog` consists of one or more (mandatory or optional) components which are specific to that catalog. A catalog should be re-usable within multiple Trino clusters. Catalogs are referenced by Trino clusters with labels and label selectors: this is consistent with the Kubernetes paradigm and keeps the definitions simple and flexible.
```

Comment:
The text explains already that catalogues are reusable, which is great. I think it would be _extra_ nice if we had an example, like for the s3 concept docs: https://docs.stackable.tech/home/stable/concepts/s3.html#_examples

You could also make a diagram, but that might be a step too much

Reply:
I've added a diagram, that looks like this (it doesn't load in my IDE):

![image](https://user-images.githubusercontent.com/1712947/191495289-ea39fcc8-af20-4c1b-9434-a3e118ac1f35.png)

Reply:
See [8882d8e](https://github.com/stackabletech/trino-operator/pull/291/commits/8882d8e17e29c719218125fa3bbfb66cb29eaf56)

Reply:
nice, looks great!